terraform/test-paas/terraform.tfvars

```diff
@@ -0,0 +1,238 @@
+credentials = {
```

Comment:
Maybe worth noting that we decided to simply copy this `tfvars` file and adjust it for the test environment rather than refactoring and using some string interpolation for the environment because this is all going to get a major refactor when we get PaaS Secrets Solution, so it didn't seem worth spending time on any refactoring now.

Reply:
Yeah agreed - will do this now